.circleci/config.yml

```diff
@@ -593,6 +596,50 @@ jobs:
           name: "Go mod tidy"
           command: make mod-tidy && git diff --exit-code
 
+  build-hive:
+    docker:
+      - image: cimg/go:1.18
```

Comment:
maybe separate, but switch to go1.19?

Reply:
Suce, I can do that. Hive builds on ancient versions of Go so it shouldn't be a big issue.